shopno/src/scripts/popUp.js

```diff
@@ -0,0 +1,55 @@
+class WorkPopup {
+    constructor(workArea) {
+        this.workArea = workArea;
+        this.addPopup();
```

Comment:
it's not supposed to be like this.. it's better to show explicitly your setup

Reply:
++++

Reply:
@Mrio1 where is update ?

Reply:
What is changed? I see different names but code is almost the same

Reply:
I tried to fix this.
```suggestion
class ImagePopup {
    constructor(parentNode) {
        this.parentNode = parentNode;
        this.addPopup();
        this.parentNode.addEventListener('click', (e) => {
            this.clickHandler(event);
        })
    }

    clickHandler({target}){
        if (target.tagName === 'IMG' && target.classList.contains('examples__image')) {
            this.activatePopup(target)
        }
    }

    addPopup() {
        this.popupNode = this.createPopupNode();
        document.body.appendChild(this.popupNode);
        this.popupImage = document.getElementById('examplesImage');
    }
```